### DISCUSSION TITLE
Change "serveless functions" folder.

### DISCUSSION THREAD

**Discussion by gabrielgrs** [vercel/vercel#5172] [Help]
Hello people!

I am starting to use vercel serveless functions and I am loving it! 
But I am not liking to use it inside "pages". 
Is there a way to change the path of the functions?

Thanks!

**Comment by styfle** [comment]
Next.js supports `pages/api` as well as `src/pages/api`.

You cannot configure a different directory name at this time.

Learn more: https://nextjs.org/docs/advanced-features/src-directory

**Reply by dbauszus-glx** [reply]
It's not possible without a framework. We also would like this. We have just a single serverless function in the project which we need to put into the api directory to make functions work. It's not the end of the world, but my OCD kicks in when I see a single file in a directory anywhere in my projects.
